redaxo/src/addons/phpmailer/install.php

```diff
@@ -28,11 +28,22 @@
     $addon->setConfig('username', '');
     $addon->setConfig('password', '');
     $addon->setConfig('smtp_debug', '0');
-    $addon->setConfig('log', 0);
+    $addon->setConfig('logging', 0);
```

Comment:
```suggestion
    $addon->setConfig('logging', false);
```

Reply:
Also mit `false`, `1`, `2` arbeiten?
Nach dem Config-Speichern würde hier wieder eine 0 draus werden:
https://github.com/redaxo/redaxo/pull/3109/files#diff-b06c2a6d051d28c72c73d379d579e76cR35

Deswegen bin ich hier bei 0 geblieben, weil ich die andere Stelle nicht umbauen wollte.

Reply:
Hmm 2 zeilen drunter arbeitest du auch mit false wenn not defined...?

Reply:
Ok, habe es dort nun auch wieder auf 0 geändert.
(Ich wollte erst zu false, und wegen der anderen stelle hatte ich es abgebrochen)